src/main/java/educonnect/logic/parser/AddCommandParser.java

```diff
@@ -84,8 +84,9 @@ public AddCommand parse(String args) throws ParseException {
      */
     public static ArrayList<String> tokenizeForTimetable(String fullTimetableString) {
         ArgumentMultimap argMultimap =
-                ArgumentTokenizer.tokenize(fullTimetableString, PREFIX_TIMETABLE_MONDAY, PREFIX_TIMETABLE_TUESDAY,
-                        PREFIX_TIMETABLE_WEDNESDAY, PREFIX_TIMETABLE_THURSDAY, PREFIX_TIMETABLE_FRIDAY,
+                ArgumentTokenizer.tokenize(" " + fullTimetableString.toLowerCase(), PREFIX_TIMETABLE_MONDAY,
```

Comment:
LGTM, nice idea to add in the space to resolve the bug and also making it case insensitive:)